### ISSUE TITLE
bone.js Pinmux Value Missing? P9_25

### ISSUE THREAD

**Issue by scirelli** [jadonk/bonescript#195]
I'm still learning BBB, device trees, how to parse the bone.js pins, etc. etc. But looking at lines [4975-4983](https://github.com/jadonk/bonescript/blob/master/src/bone.js#L4975-L4983) the options seem to be missing mode 5 & 6 which is listed in a few different references one being [this one](https://ofitselfso.com/BeagleNotes/BeagleboneBlackPRUInputPinMuxModes.pdf). I know there's a mode 6, I can set it and use it.

`pr1_pru0_pru_r31_7`
`pr1_pru0_pru_r30_7`

**Comment by jadonk** [comment]
Yeah, it isn't complete for all the modes and this project has mostly gone dormant as people program BeagleBone with other approaches. I suggest looking at bb-config: https://docs.beagleboard.org/latest/projects/bb-config/index.html
